pkg/assembler/backends/inmem/pkg.go

```diff
@@ -17,9 +17,11 @@ package inmem
 
 import (
 	"context"
+	"crypto/sha256"
 	"errors"
 	"fmt"
 	"reflect"
+	"slices"
```

Comment:
CI seems to fail here with 

```
package slices is not in GOROOT (/opt/hostedtoolcache/go/1.19.13/x64/src/slices)
```

Might need to first see how to fix that (separate PR) before we can land this PR